pkgs/top-level/qt5-packages.nix

```diff
@@ -235,5 +235,7 @@ in (kdeFrameworks // plasmaMobileGear // plasma5 // plasma5.thirdParty // kdeGea
 
   soundkonverter = callPackage ../applications/audio/soundkonverter {};
 
+  xp-pen-deco-01-v2-driver = callPackage ../os-specific/linux/pentablet-drivers/xp-pen-deco-01-v2-driver {};
```

Comment:
In retrospect i feel a better name for the folder would be `xp-pen-drivers`, that way we can remove the `xp-pen` part from the individual packages folders. Also, `{}` -> `{ }`.